### PR TITLE
Nfdiv-1968 Update jurisdiction content

### PR DESCRIPTION
**Pull request by lara-mpt**
### JIRA link ###
https://tools.hmcts.net/jira/browse/NFDIV-1968

### Change description ###
Update jurisdiction content

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/app/jurisdiction/bulletedPointsContent.ts [hunk 1]
```diff
@@ -12,69 +12,76 @@ export const enConnectionBulletPointsSummarisedForAllUsers = (
...
-    [JurisdictionConnections.APP_1_RESIDENT_TWELVE_MONTHS]: `${
-      isJointApplication ? 'applicant 1' : 'the applicant'
-    } is habitually resident in England and Wales and has resided there for at least a year immediately prior to the presentation of the application`,
-    [JurisdictionConnections.APP_1_RESIDENT_SIX_MONTHS]: `${
-      isJointApplication ? 'applicant 1' : 'the applicant'
-    } is domiciled and habitually resident in England and Wales and has resided there for at least six months immediately prior to the application`,
-    [JurisdictionConnections.APP_1_APP_2_DOMICILED]: `${isJointApplication ? 'applicant 1' : 'the applicant'} and ${
-      isJointApplication ? 'applicant 2' : 'the respondent'
-    } are both domiciled in England and Wales`,
+    [JurisdictionConnections.APP_2_RESIDENT_SOLE]: 'the respondent is habitually resident in England and Wales',
```
Comment (danielbe11) [inline src/main/app/jurisdiction/bulletedPointsContent.ts:15]:
```suggestion
    [JurisdictionConnections.APP_2_RESIDENT_SOLE]: 'the respondent is habitually resident in England and Wales.',
```

Comment (lara-mpt) [reply src/main/app/jurisdiction/bulletedPointsContent.ts:15]: @danielbe11 there shouldn't be any full stops but I've double-checked and I've left a couple in, so I'll remove those

File: src/main/app/jurisdiction/bulletedPointsContent.ts [hunk 2]
```diff
@@ -12,69 +12,76 @@ export const enConnectionBulletPointsSummarisedForAllUsers = (
...
+    [JurisdictionConnections.APP_2_RESIDENT_JOINT]: 'applicant 2 is habitually resident in England and Wales.',
+    [JurisdictionConnections.APP_1_RESIDENT_TWELVE_MONTHS]:
+      'the applicant is habitually resident in England and Wales ' +
+      'and has resided there for at least one year immediately before the application was made.',
+    [JurisdictionConnections.APP_1_RESIDENT_SIX_MONTHS]:
+      'The applicant is domiciled and habitually resident in England ' +
```
Comment (danielbe11) [inline src/main/app/jurisdiction/bulletedPointsContent.ts:21]:
```suggestion
      'the applicant is domiciled and habitually resident in England ' +
```

File: src/main/steps/applicant1/you-can-use-english-welsh-courts/content.ts [hunk 3]
```diff
@@ -80,25 +94,30 @@ export const form: FormContent = {
...
+      filters.push(removeConnectionsJC2);
+    } else {
+      filters.push(removeConnectionC);
     }
-
-    if (userCase.divorceOrDissolution === DivorceOrDissolution.DIVORCE && userCase.sameSex !== Checkbox.Checked) {
+    if (userCase.divorceOrDissolution === DivorceOrDissolution.DIVORCE) {
       filters.push(removeConnectionI);
     }
+    if (userCase === DivorceOrDissolution.DISSOLUTION || userCase.sameSex !== Checkbox.Checked) {
```
Comment (danielbe11) [inline src/main/steps/applicant1/you-can-use-english-welsh-courts/content.ts:118]:
```suggestion
    if (userCase.divorceOrDissolution === DivorceOrDissolution.DISSOLUTION || userCase.sameSex !== Checkbox.Checked) {
```

File: src/main/app/jurisdiction/connections.ts [hunk 4]
```diff
@@ -71,7 +71,8 @@ export const allowedToAnswerResidualJurisdiction = (
     (data.sameSex === Checkbox.Checked || data.divorceOrDissolution === DivorceOrDissolution.DISSOLUTION) &&
     data.bothLastHabituallyResident === YesOrNo.NO &&
     (!previousConnectionMadeUptoLastHabituallyResident(data, connections) ||
-      (connections.length === 1 && connections.includes(JurisdictionConnections.RESIDUAL_JURISDICTION)))
+      (connections.length === 1 && connections.includes(JurisdictionConnections.RESIDUAL_JURISDICTION_CP)) ||
```
Comment (danielbe11) [inline src/main/app/jurisdiction/connections.ts:74]: Should all be in the brackets

**Review by danielbe11 — APPROVED** [review]

**Pull request merged**